source/_integrations/reolink.markdown

```diff
@@ -412,6 +412,11 @@ Set up the Reolink integration in Home Assistant using the credentials you set i
 1. One way to show the camera in the dashboard is by using the picture glance card.
 2. For example, you can place arrow buttons on the card to [control the camera](/dashboards/picture-glance/#creating-a-card-to-control-the-camera).
 
+## Known limitations
+
+- 2-way audio or Text To Speech is not yet implemented.
+- The 4K camera streams are H265 encoded, support for playing H265 encoding in browsers (Chrome/Firefox/Edge/Safari etc.) is still very limited, therefore the 4K clear stream may not play in all browsers or on all phones in the Home Assistant Companion app. By default only the lower resolution Fluent camera entity is enabled in Home Assistant. This Fluent stream is H264 encoded and will play on any browser or phone.
```

Comment:
```suggestion
- 2-way audio or Text-to-speech is not yet implemented.
- The 4K camera streams are H265 encoded, support for playing H265 encoding in browsers (Chrome/Firefox/Edge/Safari etc.) is still very limited, therefore the 4K clear stream may not play in all browsers or on all phones in the Home Assistant Companion app. By default, only the lower-resolution Fluent camera entity is enabled in Home Assistant. This Fluent stream is H264 encoded and will play on any browser or phone.
```

About the first point: "is not yet implemented" kinda sets the expectation is would be implemented? While the reality is... it can't be implemented in Home Assistant at this moment in time. I would reword it in a way that it doesn't set future expectations.

Reply:
Well there is already a open PR that implements 2-way audio:
https://github.com/home-assistant/core/pull/127851
https://github.com/home-assistant/frontend/pull/22239

It was already tested to also work with the reolink doorbell.
I just did not have time yet to help push those PRs further.

Text to speech will be harder since I don't know of a good entity that provides a audio file wich I can then send to the camera. So that indeed will not get implemented any time soon.

Anyway your point is valid, so I will re-word.

Reply:
Changed to `2-way audio or Text-to-speech is not available in the Reolink Home Assistant integration.`